src/pages/staking/index.tsx

```diff
@@ -27,196 +25,197 @@ import StakingCommunityCallout from "../../components/Staking/StakingCommunityCa
-      "pool-risks"
-      "pool-reqs"
-      "pool-cta";
-  }
+const ContentContainer = (props: { children: ReactNode; id: string }) => {
+  return (
+    <Flex
+      flex="1 1"
+      flexBasis={theme.breakpoints.md}
+      position="relative"
+      padding={{ base: 0, md: "0 2rem 2rem" }}
+      gap={8}
+      direction="column"
+      alignItems="center"
+      sx={{
+        "h2:first-of-type, & > div:first-of-type": {
+          mt: 0,
+          pt: 0,
+        },
+      }}
+      {...props}
+    ></Flex>
+  )
+}
 
-  h4 {
-    color: #787878;
+const ComparisonGrid = (props: ChildOnlyProp) => {
+  const gridAreas = {
+    base: `"solo-title"
+        "solo-rewards"
+        "solo-risks"
+        "solo-reqs"
+        "solo-cta"
+        "saas-title"
+        "saas-rewards"
+        "saas-risks"
+        "saas-reqs"
+        "saas-cta"
+        "pool-title"
+        "pool-rewards"
+        "pool-risks"
+        "pool-reqs"
+        "pool-cta";`,
+    lg: `"solo-title saas-title pool-title"
+        "solo-rewards saas-rewards pool-rewards"
+        "solo-risks saas-risks pool-risks"
+        "solo-reqs saas-reqs pool-reqs"
+        "solo-cta saas-cta pool-cta"`,
   }
-`
+  return (
+    <Grid
+      columnGap={12}
+      gridAutoRows="minmax(64px, auto)"
+      gridTemplateColumns={{ base: "1fr", lg: "repeat(3, 1fr)" }}
+      gridTemplateAreas={gridAreas}
+      sx={{
+        h4: {
+          color: "#787878",
+        },
+      }}
+      {...props}
+    ></Grid>
+  )
+}
 
-const ColorH3 = styled.h3<{ color: string }>`
-  grid-area: ${({ color }) => {
-    switch (color) {
-      case "stakingGold":
-        return "solo-title"
-      case "stakingGreen":
-        return "saas-title"
-      case "stakingBlue":
-        return "pool-title"
-      default:
-        return ""
-    }
-  }};
-  color: ${({ theme, color }) => theme.colors[color]};
-`
+const ColorH3 = (props: { color: string; id: TranslationKey }) => (
+  <Heading as="h3" fontSize="2xl" color={props.color}>
+    <Translation id={props.id} />
+  </Heading>
+)
 
-const StyledButtonLink = styled(ButtonLink)`
-  @media (max-width: ${({ theme }) => theme.breakpoints.s}) {
-    width: 100%;
-  }
-`
-
-const CardGrid = styled.div`
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  gap: 2rem;
+const StyledButtonLink = (props: { to: string; id: TranslationKey }) => {
+  return (
+    <ButtonLink to={props.to} sx={{ width: { base: "100%", sm: "initial" } }}>
+      <Translation id={props.id} />
+    </ButtonLink>
+  )
+}
 
-  @media (max-width: ${({ theme }) => theme.breakpoints.xl}) {
-    grid-template-columns: 1fr;
-  }
-  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
-    grid-template-columns: repeat(3, 1fr);
-  }
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    grid-template-columns: 1fr;
-  }
-`
+const CardGrid = (props: ChildOnlyProp) => (
+  <Grid
+    gap={8}
+    templateColumns={{ base: "1fr", lg: "repeat(3, 1fr)" }}
```

Comment:
Original grid was:
S to M: 1fr / M to L (768 - 1024px): 3fr / L to XL (1024px - 1440px): 1fr / XL+: 3fr
Suggested new grid as below because of breakpoint changes
base to LG: 1fr; LG+ (992px+): 3fr